index.html

```diff
@@ -6,170 +6,213 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portofolio</title>
     <link rel="stylesheet" href="style.css">
-    </head>
+</head>
 <body>
-<section class="Workssection1">
+<section class="Workssection">
     <header class="Header">
         <a class="mylogo">MY logo</a>
-         <nav>
-            <img src="./image/logobuger.JPG">
-         </nav>
+        <nav class="headingmenu">
+            <a href="#"> <img href src="./image/logobuger.JPG" class="humbergor-icon"></a>
+            <div>
+               <ul class="meanmenu">
+                    <li><a href="#"><img src=" " alt="">Portoflilyo</a></li>
+                    <li><a href="#"><img src=" " alt="">About</a></li>
+                    <li><a href="#"><img src=" " alt="">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
     </header>
-     <div class="cardBack01">
-        <div>
-            <h1 class="main-header"> I'm Mihertab Glad to see you!</h1>
-            <p class="section1"> I’m a software developer! I can help you build a product , 
-                feature or website Look through some of my work and experience! If you like what 
-                you see and have a project you need coded, don’t hestiate to contact me. </p>
-        </div>
-        <div class = "social-section">
-                    <h4 class="connectmywork"> LET’S CONNECT</h4>
+    <div class="cardBackmain">
+        
+            <div>
+                <h1 class="main-header"> I'm Mihertab Glad to see you!</h1>
+                <p class="section1"> I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! If you like what 
```

Comment:
Great job working on your headline section! 💯 However, there are still some issues that must be solved in order to have a gorgeous portfolio to showcase your projects! 🤓 

- [ ] Please, consider fixing the height of the section, the template is showing that this section covers the entire screen, a good way to achieve this is by setting the height to `100vh` but remember to subtract the navbar height! 
💡 _Hint: you could use the [CSS's calc function](https://developer.mozilla.org/en-US/docs/Web/CSS/calc): `height: calc(100vh - [Navbar height goes here]);`_

- [ ] Please, consider fixing the size of the background image, at the moment it doesn't cover the entire section.
💡 _Hint: you could use the [CSS's `background-size` property](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size) for this and set it to "cover": `background-size: cover;`_

- [ ] Please, consider fixing the alignment of the items to the left as shown in the Figma template! 🎨 
💡 _Hint: you could use Flexbox to achieve this one!_

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201486607-fda0acff-bb3d-42b6-b7dd-35abbc06e575.png) | ![image](https://user-images.githubusercontent.com/77069011/201486577-968a7aba-3968-434f-8a74-f4f8bcea8950.png) |

Please implement these changes! they will improve the look and feel of your portfolio! 🤓 🛠️